velox/common/memory/MemoryAllocator.h

```diff
@@ -311,6 +311,47 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
   /// multiple of 'alignmentBytes'.
   static void alignmentCheck(uint64_t allocateBytes, uint16_t alignmentBytes);
 
+  /// Causes 'failure' to occur in memory allocation calls. This is a test-only
+  /// function for validating otherwise unreachable error paths. If 'persistent'
```

Comment:
> unreachable

Are those unreachable or hard-to-reach? Would be nice to clarify.

Reply:
It mean that it is very rare condition that is hard to trigger in unit test.